src/darwin/Framework/CHIP/MTRCSRInfo.h

```diff
@@ -20,9 +20,31 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- * Represents information relating to NOC CSR.
+ * Represents information relating to a certificate signing request for a Matter
+ * operational certificate.  The entire TLV nocsr-elements structure, including
+ * any vendor-specific data, is included, as well as the parsed-out CSR and
+ * CSRNonce fields from that structure.
  *
+ * The AttestationSignature from the CSRResponse command is also included.  This
+ * signs a concatenation of csrElements and the attestationChallenge found in
+ * MTRAttestationInfo.
  */
+MTR_NEWLY_AVAILABLE
+@interface MTROperationalCSRInfo : NSObject
+
+@property (nonatomic, copy, readonly) NSData * csr;
```

Comment:
Should this be `MTRCertificateDERBytes` ? I guess it's not strictly a certificate, but it's DER...

Reply:
For certs the key is to distinguish DER-encoded certs from TLV-encoded certs.  For the CSR, we don't ever have a TLV-encoded CSR floating around so there isn't any confusion....

Reply:
Added an `MTRCSRDERBytes` typedef.